packages/edit-site/src/components/layout/index.js

```diff
@@ -231,3 +233,11 @@ export default function Layout( { route } ) {
 		</>
 	);
 }
+
+export default function LayoutWithGlobalStylesProvider( props ) {
+	return (
+		<GlobalStylesProvider>
```

Comment:
Would love if we can get rid of "GlobalStylesProvider" at some point. It feels like something that should be absorbed into `core-data` at some point.